tests/test_core.py

```diff
@@ -351,6 +351,10 @@ def test_hello_world(self):
     src = open('src.c.o.js').read()
     assert 'EMSCRIPTEN_GENERATED_FUNCTIONS' not in src, 'must not emit this unneeded internal thing'
 
+  @also_with_standalone_wasm
+  def test_hello_argc(self):
+    self.do_run_in_out_file_test('tests', 'core', 'test_hello_argc')
```

Comment:
is a new test needed? a bunch of other tests happen to use argc/argv.

Reply:
Yeah, I wanted a small targeted one that runs in standalone mode.  Maybe unnecessary, but this helped me while writing this patch.